lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -44,23 +44,28 @@ class DecidirGateway < Gateway
       def initialize(options={})
         requires!(options, :api_key)
         super
+        @options[:two_phase_capture] ||= false
```

Comment:
@curiousepic Open to suggestions on naming. This is the flag to determine single phase (purchase) vs. two phase (auth + capture).

Reply:
The term "Preauth" to refer to auth+capture is more common. Since the default is one-phase, the flag might simply be called "Preauth" or "Preauth_mode" (I came around on "mode').

Reply:
It's also usually good to reflect the gateway's particular nomenclature - exactly how do they refer to this mode?

Reply:
I wasn't sure if `preauth_*` would be descriptive enough to match the docs. They gateway refers to this as `Transactions in two steps` (untranslated `Transacciones en Dos Pasos`). Aka pre-authorization and capture/confirmation. I'm ok changing this to `preauth_mode`.

Reply:
@jknipp Yeah I think that's probably most suitable.